packages/buffetjs-core/src/components/TableHeader/index.js

```diff
@@ -12,11 +12,11 @@ import Icon from '../Icon';
 
 /* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
 function TableHeader({
+  addCell,
```

Comment:
Can you keep the verb "shouldAddCell" at the beginning of the variable name, please. This is a naming rule for booleans props.